projects/schematics/src/shared/utils/file-utils_spec.ts

```diff
@@ -432,28 +432,25 @@ describe('File utils', () => {
       });
       it('should commit provided ReplaceChange', async () => {
         const filePath = '/src/app/app.component.ts';
-        const change = 'ChangedAppComponent';
+        const original = 'AppComponent';
+        const change = 'ChangedComponent';
 
-        const testChange = new ReplaceChange(
-          filePath,
-          173,
-          'AppComponent',
-          change
-        );
+        const testChange = new ReplaceChange(filePath, 192, original, change);
```

Comment:
[QUESTION]
Just to better understand: why we had to change those original numbers in this file? 173, 188, 158, 171

Reply:
Yes, it is - change is required due to the `standalone: false` in every component/directive/pipe decorator